.pre-commit-config.yaml

```diff
@@ -5,6 +5,26 @@ repos:
       - id: prettier
         additional_dependencies:
           - prettier@3.6.2
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+        exclude: |
+          (?x)^(
+              modules/nf-core/.*|
+              subworkflows/nf-core/.*|
```

Comment:
we don't want to exclude the modules and subworkflows files here, no? We exclude them in the template because they are tested in the modules repo.

Reply:
TOTALLY right